src/main/scala/sangria/slowlog/QueryMetrics.scala

```diff
@@ -168,23 +168,24 @@ case class QueryMetrics(
       typeInfo: TypeInfo,
       typeMetrics: TrieMap[String, FieldMetrics],
       prefix: String = "")(implicit renderer: MetricRenderer): Vector[ast.Comment] = {
+    val previousParentType = typeInfo.previousParentType
```

Comment:
found during profiling, `typeInfo.previousParentType` is actually an expensive `def`